pkg/xray/resource_xray_license_policy.go

```diff
@@ -149,9 +152,10 @@ func resourceXrayLicensePolicyV2() *schema.Resource {
 										Default:  false,
 									},
 									"custom_severity": {
-										Type:     schema.TypeString,
-										Optional: true,
-										Default:  "High",
+										Type:             schema.TypeString,
+										Optional:         true,
+										Default:          "High",
+										ValidateDiagFunc: validation.ToDiagFunc(validation.StringInSlice([]string{"Critical", "High", "Medium", "Low"}, true)),
```

Comment:
I believe there is also "Unknown"

Reply:
No. The default is "High" and there are no other options. 
![image](https://user-images.githubusercontent.com/55418063/144499336-48645858-4d96-4bcf-814b-6726add86ccf.png)